src/app/app.js

```diff
@@ -24,6 +24,8 @@
     hljsServiceProvider.setOptions({
       classPrefix: 'raml-console-hljs-'
     });
+  }).run(function($window) {
```

Comment:
Won't this automatically break any deployed version of API console in the wild, by changing the default?